ui/pages/destinations/add.js

```diff
@@ -97,7 +97,7 @@ export default function DestinationsAdd() {
         extensionDeadline: '720h',
       }),
     })
-    const key = await res.json()
```

Comment:
what is` jsonBody()`? what is that difference between `res.json()` and `jsonBody(rest) `?

Reply:
see the implementation here https://github.com/infrahq/infra/pull/3446/files#diff-350a2c2ecbe2db05755f218618618e394994910966e81d4fed13d9d4689e646c